### PR TITLE
Install a .desktop and icon files on Unix

### PR DESCRIPTION
**Pull request by linkmauve**
The icon is the same PNG as the one embedded in [data/windows/icon.ico](https://github.com/Gargaj/Bonzomatic/blob/55db10f504600a7045bca8761d57f09e8d1f595a/data/windows/icon.ico), losslessly recompressed using [oxipng](https://github.com/shssoichiro/oxipng).

I also renamed the binary to be closer to the expected pattern on this platform.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,11 @@ cmake_minimum_required(VERSION 3.0)
 set(CMAKE_OSX_ARCHITECTURES x86_64)
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
-set(BZC_EXE_NAME "Bonzomatic")
+if (UNIX AND (NOT APPLE))
+  set(BZC_EXE_NAME "bonzomatic")
```
[the remark this answers is not in the document]
Comment (linkmauve) [reply CMakeLists.txt]: The expected binary name is lowercase on Linux systems and similar.

File: CMakeLists.txt [hunk 2]
```diff
@@ -502,4 +506,9 @@ if (UNIX AND (NOT APPLE))
     install(FILES "${CMAKE_SOURCE_DIR}/external/bass/x86/libbass.so" DESTINATION lib)
   endif ()
   install(TARGETS ${BZC_EXE_NAME} RUNTIME DESTINATION bin)
+  install(FILES "${CMAKE_SOURCE_DIR}/data/unix/bonzomatic.desktop"
```
[the remark this answers is not in the document]
Comment (linkmauve) [reply CMakeLists.txt]: No, the .desktop format is a standard (see the [desktop entry specification](https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html)) which works on all distributions, and on most desktop environments.

File: CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gargaj) [reply CMakeLists.txt]: Then set everything to lowercase (Windows has the common sense to be case-insensitive so noone really cares), but don't make platform-specific hacks for what are essentially aesthetics.

[the remark this answers is not in the document]
Comment (linkmauve) [reply CMakeLists.txt]: I’ve done that in the latest version.

**Comment by alkama** [comment]
So thank you, now the mac version has a lowercase .app
Quite ugly :D

**Pull request merged**